lib/actions/FrontendAction.js

```diff
@@ -60,6 +71,24 @@ class FrontendAction {
       }
     }
   }
+
+  themeChanged (dcClient, config, cb = () => {}) {
+    console.log(1)
+    async.retry({
+      times: 5,
+      interval: (count) => 100 * Math.pow(2, count)
+    }, (acb) => {
+      dcClient.generateExtensionConfig(config.file, AppSettings.getInstance().getId(), (err, extConfig) => {
+        if (err) return acb(new Error(`Could not generate Config for '${config.file.id}'`))
+        return acb(null, extConfig)
+      })
+    }, (err, extConfig) => {
+      if (err) return
+      fsEx.outputJsonSync(path.join('', config.path, 'config', 'app.json'), extConfig, {spaces: 2})
```

Comment:
Currently the app.json looks like:
```json
{
  "frontend": {
    "language": "en-us",
    "publicPath": "http://dgthqh2gka0ky.cloudfront.net/shop_30685/@shopgate/theme-gmd/0.1.21",
    "appId": "shop_30685",
    "logo": "https://data.shopgate.com/shops/24327/webapp@2x.png",
    "colors": {},
    "shopName": "sgxs-testshop-us1",
    "forgotPasswordUrl": null,
    "webCheckoutShopify": null,
    "features": {
      "showReviews": true
    }
  },
  "warnings": [
    "Using default ({}) for key 'colors'",
    "Using default (null) for key 'forgotPasswordUrl'",
    "Using default (null) for key 'webCheckoutShopify'"
  ]
}
```

But it should only contain the content of the frontend property